app/src/main/res/navigation/setup_activity_nav_graph.xml

```diff
@@ -35,7 +35,7 @@
     app:layout="@layout/screen_onboarding_old"
     tools:layout="@layout/screen_onboarding_old">
     <action
-      android:id="@+id/action_onboardingScreen_to_selectCountryScreen"
+      android:id="@+id/action_onboardingScreenOld_to_selectCountryScreen"
```

Comment:
Let's rename this in earlier commit where we are renaming `OnboardingScreen` to `OnboardingScreen_Old`